samples/dashboards/sync/custom-component-sync/demo.js

```diff
@@ -0,0 +1,369 @@
+        }
+
+        this.cell.setLoadingState(false);
+    }
+}
+
+ComponentRegistry.registerComponent('Slider', Slider);
+
+const formatBigNumber = value => {
+    if (value >= 1e6) {
+        return Math.round(value / 1e5) / 10 + 'M';
+    }
+    if (value >= 1e3) {
+        return Math.round(value / 1e3) + 'k';
+    }
+    return value;
+};
+
+Dashboards.board('container', {
+    gui: {
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'chart-cell'
+                }, {
+                    id: 'slider-cell'
+                }]
+            }]
+        }]
+    },
+    dataPool: {
+        connectors: [{
+            id: 'data',
+            type: 'CSV',
+            options: {
+                csv: document.getElementById('csv').innerHTML
+            }
+        }]
+    },
+    components: [{
+        renderTo: 'slider-cell',
+        type: 'Slider',
+        title: {
+            text: 'Population'
+        },
+        connector: {
+            id: 'data',
+            valuesColumn: 'Population'
+        },
+        valueFormatter: value => formatBigNumber(value),
+        sync: {
+            customSync: {
+                enabled: true,
+                emitter: function () {
+                    const { board } = this;
+                    const { dataCursor: cursor } = board;
+
+                    if (
+                        !board ||
+                        !this.sync.syncConfig.customSync.enabled
+                    ) {
+                        return;
+                    }
+
+                    return this.on('sliderValueChanged', e => {
+                        const table = this.connector && this.connector.table;
+                        if (table) {
+                            // Emit cursor event when slider value changes
+                            cursor.emitCursor(table, {
+                                type: 'position',
+                                row: e.rowIndex,
+                                target: this,
+                                state: 'sliderComponent.move'
+                            });
+                        }
+                    });
+                }
+            }
+        }
+    }, {
+        renderTo: 'chart-cell',
+        type: 'Highcharts',
+        connector: {
+            id: 'data'
+        },
+        columnAssignment: {
+            Population: 'x',
+            'Happiness vs Population': {
+                y: 'Happiness',
+                name: 'Country'
+            }
+        },
+        chartOptions: {
+            xAxis: {
+                type: 'logarithmic',
+                title: {
+                    text: 'Population'
+                }
+            },
+            yAxis: {
+                title: {
+                    text: 'Happiness'
+                }
+            },
+            title: {
+                text: 'Happiness vs Population'
```

Comment:
Change to "Happiness Score vs Population YYYY", where is YYYY is the correct year,